### PR TITLE
Dev k tk064

### PR DESCRIPTION
**Pull request by kailasb**
added functionality for send email for corporate lead.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,8 @@
...
 /media/
-app/config/db.php
-app/config/config.php
-app/config/social.php
+/app/config/db.php
+/app/config/config.php
+/app/config/social.php
+/app/libraries/batcave.php
+/app/libraries/loader.php
+.htaccess
```
Comment (medini-ddl) [inline .gitignore:8]: Its not good practice to add loader file in gitignore
batcave.php has other application based functions and need not be included here.

Remove them

File: app/tasks/leads/compiled2leadsquare.php [hunk 2]
```diff
@@ -102,3 +108,20 @@ function compiledLeadCronfailure($errorFlag = '', $cronTracker) {
...
     }
 }
+
+/*
+*
+*/
+function send_corporate_lead_email($compiledLead)
+{
+	//echo 'test locally corporate email called.=>';
+	//require_once('testemail.php');
```
Comment (medini-ddl) [inline app/tasks/leads/compiled2leadsquare.php:118]: Remove commented codes

File: app/tasks/leads/compiled2leadsquare.php [hunk 3]
```diff
@@ -102,3 +108,20 @@ function compiledLeadCronfailure($errorFlag = '', $cronTracker) {
         exit();
     }
 }
+
+/*
+*
+*/
+function send_corporate_lead_email($compiledLead)
```
Comment (medini-ddl) [inline app/tasks/leads/compiled2leadsquare.php:115]: Its standard suggestion to use camelcase function names

File: .gitignore (hunk 1 above)
Comment (kailasb) [reply .gitignore]: as I'm not aware about the standards. as make changes while setup locally. that's why added in gitignore file & it should not affect on dev.

so as you mentioned the changes I've reverted the gitignore file.

File: app/tasks/leads/compiled2leadsquare.php (hunk 2 above)
Comment (kailasb) [reply app/tasks/leads/compiled2leadsquare.php]: Removed the commented code

File: app/tasks/leads/compiled2leadsquare.php (hunk 3 above)
Comment (kailasb) [reply app/tasks/leads/compiled2leadsquare.php]: changed the function name by using camelcase standard of function name.

**Pull request merged**